nav2_system_tests/src/planning/planner_tester.cpp

```diff
@@ -440,7 +491,7 @@ bool PlannerTester::sendCancel()
   return false;
 }
 
-void PlannerTester::printPath(const nav2_behavior_tree::ComputePathToPoseResult & path) const
+void PlannerTester::printPath(const ComputePathToPoseResult & path) const
```

Comment:
you could consider using a `std::stringstream` instead of printing multiple `RCLCPP_INFO` lines